distributed/protocol/compression.py

```diff
@@ -93,6 +93,41 @@ def _fixed_lz4_decompress(data):
     }
     default_compression = "lz4"
 
+with ignoring(ImportError):
+    _import_policy = dask.config.get("distributed.comm.zstd.import_policy", "default")
+    os.environ["PYTHON_ZSTANDARD_IMPORT_POLICY"] = _import_policy
```

Comment:
Why this change?

Reply:
according to:     [Zstandard](https://pypi.org/project/zstandard/) :
zstandard provided some module import policy: use "default", "cffi_fallback", "cext", or "cffi" ;

and as a public library implementation, we let user make their best choice.

Reply:
The user can set this environment variable independently.  I am not so sure that it is a good idea to do this for them.

Reply:
cc @jcrist, thoughts on things like this?

Reply:
Yeah, the user should set this themselves if they care, distributed shouldn't modify environment variables like this.

Reply:
Please remove these lines.